packages/module/src/catalog/Toolbar/QuickStartCatalogFilterItems.tsx

```diff
@@ -72,7 +72,7 @@ interface QuickStartCatalogFilterSearchWrapperProps {
   onSearchInputChange: any;
```

Comment:
You want change this here:

```suggestion
  onSearchInputChange: (val: string);
```